Tests/Common/Orders/Fills/ImmediateFillModelTests.cs

```diff
@@ -39,7 +39,7 @@ public void PerformsMarketFillBuy()
         {
             var model = new ImmediateFillModel();
             var order = new MarketOrder(Symbols.SPY, 100, Noon);
-            var config = CreateTradeBarConfig(Symbols.SPY);
+            var config = CreateQuoteBarConfig(Symbols.SPY);
```

Comment:
I don't think these unit tests are performing as expected, most of them are setting a `new TradeBar` into the security cache so there is no quote bar to use and uses the trade bar instead. Also, I think we need *new* tests for QuoteBar configurations but keeping existing tests too?